### DISCUSSION TITLE
Adjust suggestions to new Memory Limit faster

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#232] [Feature Requests]
Now, It takes several days to adjust suggestions to new Memory Limit.

**Comment by kochetovd** [comment]
Now, Releem will to adjust suggestions within 12 hours when memory limit is reduced

**Comment by drupaladmin** [comment]
clarity of what’s happening and clarity of what “settings” are assumed for the config , will be useful to give users more confidence in deciding to apply the config

**Comment by drupaladmin** [comment]
Improved in the [1.19.8](https://github.com/Releem/mysqlconfigurer/releases/tag/1.19.8)
